source/controlTypes/state.py

```diff
@@ -71,6 +71,7 @@ def negativeDisplayString(self) -> str:
 	OVERFLOWING = 0x10000000000
 	UNLOCKED = 0x20000000000
 	HAS_ARIA_DETAILS = 0x40000000000
+	HASNOTE = 0x80000000000

```

Comment:
I believe the object model impl of Excel only supports notes, previously referred to as comments. Could you swap states in there as well, or should this be done in a follow up?

Reply:
I think if we do it at all, I'd prefer to do it in a follow up. It touches completely different code.